code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -58,9 +58,9 @@
 
 /obj/item/weapon/melee/energy/suicide_act(mob/user)
 	var/tempgender = "[user.gender == MALE ? "he's" : user.gender == FEMALE ? "she's" : "they are"]"
-	if (active)
-		viewers(user) << pick("<span class='danger'>\The [user] is slitting \his stomach open with \the [src]! It looks like [tempgender] trying to commit seppuku.</span>", \
-		                      "<span class='danger'>\The [user] is falling on \the [src]! It looks like [tempgender] trying to commit suicide.</span>")
+	if(active)
+		to_chat(viewers(user), pick("<span class='danger'>\The [user] is slitting \his stomach open with \the [src]! It looks like [tempgender] trying to commit seppuku.</span>", \
+		                      		"<span class='danger'>\The [user] is falling on \the [src]! It looks like [tempgender] trying to commit suicide.</span>"))

```

Comment:
Think the indentation mess-up is somewhere around here. No idea how to format this, though.

Reply:
Replace that to_chat with visible_message, maybe?